bin/agentConsole.js

```diff
@@ -27,7 +27,7 @@ var readline = require('readline'),
     iotAgentLib = require('../lib/fiware-iotagent-lib'),
     commandUtils = require('command-node'),
     config = require('../config'),
-    logger = require('fiware-node-logger'),
+    logger = require('logops'),
```

Comment:
I may have observed (I'm not sure) that the `logops` module does not respond correctly when changing the logging level ("DEBUG", "INFO", etc.). Although not related to this PR, would you mind trying to change the logging level to check if the logging behaves correctly, please :)